docker-compose.yml

```diff
@@ -9,9 +8,7 @@ services:
         environment:
             - JWT_ENABLED=false
             - ALLOW_PRIVATE_IP_ADDRESS=true
-    ds-test-example:
+    example:
```

Comment:
Do we need change that name? I'm not 100% that we don't have other project rely on that name, probably then we check compliation of DocumentServer from the source code

Reply:
Maybe `node_example`?

Reply:
Maybe leave the original name? Are there problems with the old name?

I think we may have some services/instructions that rely on the old name
Of course, we can change it, but what's the point?